pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -374,6 +385,59 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
 	return nil
 }
 
+// newExternalCephObjectStoreInstances returns a set of CephObjectStores
+// needed for external cluster mode
+func (r *ReconcileStorageCluster) newExternalCephObjectStoreInstances(
+	initData *ocsv1.StorageCluster, rgwEndpoint string, reqLogger logr.Logger) ([]*cephv1.CephObjectStore, error) {
+	// check whether the provided rgw endpoint is empty
+	if rgwEndpoint = strings.TrimSpace(rgwEndpoint); rgwEndpoint == "" {
+		reqLogger.Info("WARNING: Empty RGW Endpoint specified, external CephObjectStore won't be created")
```

Comment:
Is it a warning or an error? Either way, use the appropriate `reqLogger`.

Reply:
Yes it is a `Warning` only and, in a normal scenario, this function won't be even called (as won't be propagating the `ceph-rgw` in the json output).
`reqLogger` (an instance of `github.com/go-logr/logr` interface) don't have a `.Warn` (and have a [philosophy](https://dave.cheney.net/2015/11/05/lets-talk-about-logging) on why no warnings). So `Info` is the best (and non-complicated) bet to go.